packages/@react-native-community/changelog-generator/src/changelog-generator.ts

```diff
@@ -506,6 +501,7 @@ export function getChangelogDesc(
   const acc = deepmerge(CHANGES_TEMPLATE, {});
   const commitsWithoutExactChangelogTemplate: string[] = [];
 
+  // eslint-disable-next-line complexity
```

Comment:
The arrow function has a large cyclomatic complexity, so VS Code with our rules marks the entire function yellow. Disabling the warning on the existing high complexity function.